Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -359,9 +362,9 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
     }
 
     private func load(options: StartupOptions) throws {
-        isConnectionTesterEnabled = options.enableTester
```

Comment:
This is no longer good since we now cancel the tunnel, and on-demand restarts it but the option won't be present in the options dictionary.

I made some change so that the option is persisted across launches, unless explicitly overridden.